docs/UserGuide/Cluster/Cluster-Setup-Example.md

```diff
@@ -65,4 +65,21 @@ nohup ./cluster/target/iotdb-cluster-0.13.0-SNAPSHOT/sbin/start-node.sh ./cluste
 nohup ./cluster/target/iotdb-cluster-0.13.0-SNAPSHOT/sbin/start-node.sh ./cluster/target/test-classes/node3conf/ >/dev/null 2>&1 &
 ```
 ### Use the official website release version:
-You can refer to the above example, but starting multiple instances on a single machine requires handling port and file directory conflicts.
\ No newline at end of file
+```
+curl -O https://mirrors.bfsu.edu.cn/apache/iotdb/0.12.0/apache-iotdb-0.12.0-cluster-bin.zip
```

Comment:
It may be better to use the package of Apache's official website

Reply:
Ok. I modify it to the package of Apache's website.